sdk/keyvault/keyvault-admin/src/generated/static-helpers/pollingHelpers.ts

```diff
@@ -113,6 +118,7 @@ export function getLongRunningPoller<
     processResult: (result: unknown) => {
       return processResponseBody(result as TResponse);
     },
+    updateState: options?.updateState
```

Comment:
Please note the change is manually updated in this helper, I plan to adopt this in codegen side.